packages/scorchwrap/src/runtime/runtime.js

```diff
@@ -57,10 +57,7 @@ const enforcePolicy = (requestedResourceId, referrerResourceId) => {
   ) {
     return
   }
-  const myPolicy = LAVAMOAT.policy.resources[referrerResourceId]
-  if (!myPolicy) {
-    throw Error('Policy missing for ' + referrerResourceId)
```

Comment:
shouldn't we strictly expect policies to always exist?

Reply:
That was my initial implementation and it's not what the rest of LavaMot is doing. Policy generation skips all packages that don't use any powers. It's necessary for eliminating a lot of spam from the policy definition.